src/librustc/session/config.rs

```diff
@@ -605,6 +605,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
           "Print the size of enums and their variants"),
     force_overflow_checks: Option<bool> = (None, parse_opt_bool,
           "Force overflow checks on or off"),
+    trace_macros: bool = (false, parse_bool,
+          "Trace all macro invocations"),
```

Comment:
Could this be a bit more descriptive? What is the user-visible effect of tracing?